recipes/conda-self/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "simplejson" %}
```

Comment:
Looks liek this `name` is never used. Can be removed

Reply:
```suggestion
```